colossalai/zero/sharded_model/_utils.py

```diff
@@ -35,7 +37,10 @@ def cast_tensor_to_fp16(tensor: torch.Tensor) -> torch.Tensor:
     return tensor
 
 
-def cast_tensor_to_fp32(tensor: torch.Tensor) -> torch.Tensor:
```

Comment:
Should we also modify `cast_tensor_fo_fp16()` for stateful tensor?

Reply:
DONE